ci-operator/config/openshift/console-plugin-template/openshift-console-plugin-template-main.yaml

```diff
@@ -1,21 +1,10 @@
-base_images:
-  nginx:
-    name: nginx
-    namespace: openshift
-    tag: stable
-  nodejs-16:
-    name: nodejs-16
-    namespace: openshift
-    tag: latest
 build_root:
-  from_repository: true
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: tectonic-console-builder-v24
```

Comment:
@spadgett Using the `image_stream_tag` with `tectonic-console-builder-v24` similar to how it is used in [kubevirt](https://github.com/openshift/release/blob/master/ci-operator/config/kubevirt-ui/kubevirt-plugin/kubevirt-ui-kubevirt-plugin-main.yaml#L2) config. Hope this is right for our use case?

Reply:
I assume this is needed for Cypress to run? We can use this for now, but it would be good to eventually move to an image with just the Cypress dependencies.